peps/converters.py

```diff
@@ -199,6 +202,28 @@ def get_pep_page(artifact_path, pep_number, commit=True):
     return pep_page
 
 
+def get_commit_history_info(pep_filename):
+    commit_info = ''
+
+    try:
+        data = requests.get(
+            'https://api.github.com/repos/python/peps/commits?path={}'.format(pep_filename)
+        )
+    except Exception:
+        return commit_info
+
+    json_data = data.json()
+    if len(json_data) > 0:
+        commit_date = json_data[0]['commit']['committer']['date']
```

Comment:
Are there any failure conditions in which this would generate a `KeyError`?

Reply:
As far as I'm aware, github's API will always return `[]` upon finding no results or error, and when results are found for the repository's commits, there will always be an accessible `commit.committer.date` field. So, the `KeyError` would not occur in any situation (that I'm aware of) where `data.json()` returns something with a length > 0 when targeted at that specific URL.